elastic/integrations
# [Hashicorp Vault] Added dimensions mapping and added missing label fields to metrics datastream

- Enhancement

## Proposed commit message

- WHAT: Added dimensions mapping to metrics datastream. Missing fields added to annotate as dimensions. 
- WHY : To benefit from the TSDB enablement of the package.



## Checklist



## How to test this PR locally

- elastic-package build
- elastic-package stack up -v -d --services package-registry
- TSDB-migration-test-kit ran to check record count validation.  Test results shared in the comments.
- 

## Related issues
- https://github.com/elastic/integrations/issues/8128 

## Screenshots

No related dashboards found
